02-Arrays/02-Substitution/substitution.c

```diff
@@ -0,0 +1,145 @@
+#include <cs50.h>
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+
+bool validate_arguments(int arguments);
+bool validate_key(string key);
+bool validate_key_length(string key);
+bool validate_key_alphabetic(string key);
+bool validate_key_unique(string key);
+char encrypt_character(char character, string key);
+
+
+int main(int argc, string argv[])
```

Comment:
Can you spot one or two logical groups to extract from the `main` function?

https://refactoring.guru/extract-method

Reply:
Hm, maybe I could group validating the arguments and the key to a single `validate_input` function.

```
if (validate_input(argc, argv[1]) != true) {
   return 1
}
```